addons/howTo/guides/rpg7-pgo.inc.sqf

```diff
@@ -0,0 +1,16 @@
+["RPG7 PGO Sight", [{
+    (_units findIf {((secondaryWeaponItems _x)#2) in ["gm_pgo7v_blk", "rhs_acc_pgo7v", "rhs_acc_pgo7v2", "rhs_acc_pgo7v3", "CUP_optic_PGO7V", "CUP_optic_PGO7V2", "CUP_optic_PGO7V3"]}) != -1
+}, {
+    "
+Squad equiped with PGO sight for RPG-7<br/><br/>
```

Comment:
Since this is just briefing format, can we include images?

Reply:
yeah, but you are somewhat limited by briefing size, maybe 400px wide
I've seen some img guides but I think they are wrong
eg https://i.imgur.com/PKxUY8k.png has the Frag on the left

Reply:
CUP has a nice guide in the field-manual
![Untitled](https://github.com/user-attachments/assets/31626be5-fde8-450a-b17b-f8d915490d2e)

this isn't really meant to replace that, 
but just be something that shows when your squad has some unusual kit